src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -203,7 +203,7 @@ protected function wrapValue($value)
             return $value;
         }
 
-        $value = $this->isReserved($value) ? Str::lower($value) : Str::upper($value);
+        $value = Str::upper($value);
```

Comment:
Yes, I understand your point. But due to oracle specific limitations like the reserved words, no autoincrement, etc.. I decided to compromise some of these like the column names so that the package will work most of the time with the framework. In cases where tables are not created via migration, the developer should then use `DB::raw()` for it to work as a fallback.